tests/test_type_aliases.py

```diff
@@ -0,0 +1,30 @@
+from typing import Dict, List
+
+import pytest
+
+from stable_baselines3.common.type_aliases import check_cast, non_null
+
+
+def test_non_null():
+    for a in (1, "a", [2]):
+        assert non_null(a) == a
+
+    with pytest.raises(ValueError):
+        non_null(None)
+
+
+def test_check_cast():
+    assert check_cast(dict, {}) == {}
```

Comment:
I think that what you actually want to do is create the object first and then check identity with "is".

Reply:
I guess, that's the actual guarantee, yes. Fixed in fe52f29d